src/DigitalSubscriberAppBanner/index.stories.tsx

```diff
@@ -28,16 +28,16 @@ export const defaultStory = (): ReactElement => {
             <>
                 <BrazeMessageComponent
                     componentName={componentName}
+                    brazeMessageProps={{
+                        header: header,
+                        body: body,
+                    }}
                     logButtonClickWithBraze={(internalButtonId) => {
                         console.log(`Button with internal ID ${internalButtonId} clicked`);
                     }}
                     submitComponentEvent={(componentEvent) => {
                         console.log('submitComponentEvent called with: ', componentEvent);
                     }}
-                    brazeMessageProps={{
-                        header: header,
-                        body: body,
-                    }}
```

Comment:
Same comment as above about moving this back just to reduce the size of the diff. No big deal obviously!